app/src/models/firebase/FirestoreCtrl.ts

```diff
@@ -43,11 +44,11 @@ export type DBComment = {
 };
 
 export type DBGroup = {
-  group_id: string;
-  group_name: string;
-  description?: string;
+  gid?: string;
+  name: string;
+  challengeTitle: string;
   members: string[];
-  creationDate?: Date;
+  updateDate: Timestamp;
```

Comment:
This should use the `Date` type in accordance with the rest of the DB types. Make sure to adapt the consequences as well.